release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1947,6 +1947,8 @@ This update fixes the problem by introducing client-go 1.22.
 
 * Previously, pod namespaces managed by CRI-O, for example network, IPC, and UTS, were not unmounted when the pod was removed. This resulted in leakage, driving the Open vSwitch CPU to 100%, which caused pod latency and other performance impacts. This has been resolved and pod namespaces are unmounted when removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2003193[*BZ#2003193*])
 
+* Moby (Docker Engine) containers are started with non-empty inheritable Linux process capabilities. This can potentially allow programs with inheritable file capabilities to elevate those capabilities to the permitted set during `execve(2)`. The workaround is to modify the entry point of a container using a utility such as `capsh(1)` to drop inheritable capabilities prior to when the primary process starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076265[*BZ#2076265*])
```

Comment:
I'm not familiar with this document, but do we refer to them as `Moby (Docker Engine) containers` throughout? It feels misleading to me, because the only containers that  can be started on 4.10 (and those that are affected by this) are created by cri-o

Reply:
also, we've reverted the issue in 4.10, but it will (for now) remain present in 4.11

Reply:
Yeah, I have same question. Since the containers are created by cri-o, I guess we should just mention containers?

Reply:
also, we've also reverted this patch in 4.11.6

Reply:
I've amended the text to use the simple  term "Containers". Resolving some merge conflicts: I will update the PR when they are done.